src/com/itacademy/eshop/services/ShopService.java

```diff
@@ -10,11 +11,12 @@ public class ShopService {
      */
 
     public Eshop createShop() {
+
         /** Should create new e-shop object and initialize it with some products.
          * Should return e-shop object.
          */
        ArrayList<Product> products = createProducts();
-        throw new UnsupportedOperationException("createShop() is not implemented yet");
+       return new Eshop ("name", products);
```

Comment:
you could save even one more like by doing this
`return new Eshop("name", createProducts());`